### PR TITLE
init data tables and migration

### PR DESCRIPTION
**Pull request by OrangeChickens**

### CODE REVIEW

File: src/entities/Orders.ts [hunk 1]
```diff
@@ -13,37 +14,37 @@ import { Users } from "./Users";
...
-  @Column("timestamp without time zone", { name: "updated_at", nullable: true })
+  @Column("timestamp without time zone", {nullable: true })
   updatedAt: Date | null;
 
-  @Column("character varying", { name: "created_at", nullable: true })
+  @Column("character varying", {nullable: true })
   createdAt: string | null;
 
   @OneToMany(() => Items, (items) => items.order)
   items2: Items[];
```
Comment (Noah02148) [inline src/entities/Orders.ts:45]: orderItems?

File: src/entities/Orders.ts [hunk 2]
```diff
@@ -13,37 +14,37 @@ import { Users } from "./Users";
...
+  @Column("integer", {nullable: true })
   sellerShopId: number | null;
 
-  @Column("double precision", { name: "amount", nullable: true, precision: 53 })
+  @Column("double precision", {nullable: true })
   amount: number | null;
 
-  @Column("json", { name: "items", nullable: true })
+  @Column("json", {nullable: true })
   items: object | null;
```
Comment (Noah02148) [inline src/entities/Orders.ts:30]: itemsJson?

File: src/entities/ShopFollowers.ts [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import { Column, Entity, JoinColumn, ManyToOne, PrimaryGeneratedColumn } from "typeorm";
+import { Shops } from "./Shops";
+
+@Entity("shop_followers", { schema: "public" })
+export class ShopFollowers {
+  @PrimaryGeneratedColumn("uuid")
+  followerId: string;
+
+  @ManyToOne(() => Shops, (shops) => shops.shopFollowers)
+  @JoinColumn([{referencedColumnName: "id" }])
```
Comment (Noah02148) [inline src/entities/ShopFollowers.ts:10]: JoinColumn里得有name吧

Comment (OrangeChickens) [reply src/entities/ShopFollowers.ts:10]: 会自动generate。

File: src/entities/Orders.ts (hunk 1 above)
Comment (OrangeChickens) [reply src/entities/Orders.ts:45]: done

File: src/entities/Orders.ts (hunk 2 above)
Comment (OrangeChickens) [reply src/entities/Orders.ts:30]: done

**Review by Noah02148 — APPROVED** [review]

**Pull request merged**